.github/workflows/release_docker.yml

```diff
@@ -26,35 +26,118 @@ on:
+        run: |
+          platform=${{ matrix.platform }}
+          echo "PLATFORM_PAIR=${platform//\//-}" >> $GITHUB_ENV
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@8e5442c4ef9f78752691e2d8f8d19755c6f78e81 # v5.5.1
+        with:
+          images: ${{ env.IMAGE_IOTA_NODE }}
+
+      - name: Login to Docker Registry
+        uses: docker/login-action@9780b0c442fbb1117ed29e0efdff1e18412f7567 # v3.3.0
+        with:
+          username: ${{ secrets.DOCKER_REGISTRY_USERNAME }}
+          password: ${{ secrets.DOCKER_REGISTRY_PASSWORD }}
+          registry: ${{ secrets.DOCKER_REGISTRY_URL }}
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@49b3bc8e6bdd4a60e6116a5414239cba5943d3cf # v3.2.0
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@c47758b77c9736f4b2ef4073d4d51994fabfe349 # v3.7.1
+
       - name: Checkout code
         uses: actions/checkout@eef61447b9ff4aafe5dcd4e0bbf5d482be7e7871 # v4.2.1
 
       - name: Read Rust toolchain version from TOML
-        id: read_toolchain
+        id: read-toolchain
         run: |
           TOOLCHAIN_VERSION=$(grep -oE 'channel = "[^"]+' ./rust-toolchain.toml | sed 's/channel = "//')
           echo "TOOLCHAIN_VERSION=${TOOLCHAIN_VERSION}" >> $GITHUB_ENV
 
       - name: Log Rust toolchain version
         run: echo "Rust toolchain version is ${{ env.TOOLCHAIN_VERSION }}"
 
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@49b3bc8e6bdd4a60e6116a5414239cba5943d3cf # v3.2.0
+      - name: Get git revision and build date
+        id: git-info
+        run: |
+          echo "GIT_REVISION=$(git describe --always --abbrev=12 --dirty --exclude '*')" >> $GITHUB_ENV
+          echo "BUILD_DATE=$(date -u +'%Y-%m-%d')" >> $GITHUB_ENV
+
+      - name: Build and push by digest
+        id: build
+        uses: docker/build-push-action@4f58ea79222b3b9dc2c8bbdd6debcef730109a75 # v6.9.0
+        with:
+          context: .
+          file: docker/iota-node/Dockerfile
+          platforms: ${{ matrix.platform }}
+          labels: ${{ steps.meta.outputs.labels }}
+          outputs: type=image,"name=${{ env.IMAGE_IOTA_NODE }}",push-by-digest=true,name-canonical=true,push=true
+          build-args: |
+            GIT_REVISION=${{ env.GIT_REVISION }}
+            BUILD_DATE=${{ env.BUILD_DATE }}
+            RUST_IMAGE_VERSION=${{ env.TOOLCHAIN_VERSION }}-bookworm
+
+      - name: Export digest
+        run: |
+          mkdir -p ${{ runner.temp }}/digests-node
+          digest="${{ steps.build.outputs.digest }}"
+          touch "${{ runner.temp }}/digests-node/${digest#sha256:}"
+
+      - name: Upload digest
+        uses: actions/upload-artifact@65c4c4a1ddee5b72f698fdd19549f0f0fb45cf08 # v4.6.0
+        with:
+          name: digests-node-${{ env.PLATFORM_PAIR }}
+          path: ${{ runner.temp }}/digests-node/*
+          if-no-files-found: error
+          retention-days: 1
+
+  merge-iota-node:
```

Comment:
Reminder to add this to the confluence page when the PR gets merged 🙏🏻

Reply:
I am still working on it